stm32_base/hal/gps_skytraq/gps_device_skytraq.c

```diff
@@ -32,16 +39,21 @@ typedef struct _BaudRateCodes{
 #define UPDATE_RATE_COUNT 		10
 #define UPDATE_RATES 			{1, 2, 4, 5, 8, 10, 20, 25, 40, 50}
 
-#define MSG_ID_ACK								0x83
-#define MSG_ID_NACK 							0x84
-#define MSG_ID_QUERY_SW_VERSION 				0x02
-#define MSG_ID_SET_FACTORY_DEFAULTS				0x04
-#define MSG_ID_SW_VERSION						0x80
-#define MSG_ID_QUERY_POSITION_UPDATE_RATE		0x10
-#define MSG_ID_CONFIGURE_POSITION_UPDATE_RATE	0x0E
-#define MSG_ID_POSITION_UPDATE_RATE 			0x86
-#define MSG_ID_CONFIGURE_SERIAL_PORT			0x05
-#define MSG_ID_CONFIGURE_NMEA_MESSAGE			0x08
+#define MSG_ID_ACK											0x83
+#define MSG_ID_NACK 										0x84
+#define MSG_ID_QUERY_SW_VERSION 							0x02
+#define MSG_ID_SET_FACTORY_DEFAULTS							0x04
+#define MSG_ID_SW_VERSION									0x80
+#define MSG_ID_QUERY_POSITION_UPDATE_RATE					0x10
+#define MSG_ID_CONFIGURE_POSITION_UPDATE_RATE				0x0E
+#define MSG_ID_POSITION_UPDATE_RATE 						0x86
+#define MSG_ID_CONFIGURE_SERIAL_PORT						0x05
+#define MSG_ID_CONFIGURE_GNSS_NAVIGATION_MODE               0x64
+#define MSG_SUBID_CONFIGURE_GNSS_NAVIGATION_MODE            0x17
+#define MSG_ID_CONFIGURE_NMEA_MESSAGE						0x08
+#define MSG_ID_CONFIGURE_MESSAGE_TYPE						0x09
+#define MSG_ID_CONFIGURE_NAVIGATION_DATA_MESSAGE_INTERVAL	0x11
+#define MSG_ID_NAVIGATION_DATA_MESSAGE						0xA8
 
```

Comment:
I proposed running everything through a code reformatter once this is committed.